app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-05-01"
+
+tabs:
+  opened:
+    type: labeled_counter
+    description: >
+      Number of opened tabs
+    send_in_pings:
+      - session_end
+    labels:
+      - context_menu
+      - tabs_dialog
+      - bookmarks
+      - history
+      - fxa_login
+      - received
+      - pre_existing
+      - browser
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  activated:
+    type: counter
+    description: >
+      Number of activated tabs
+    send_in_pings:
+      - session_end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+firefox_account:
+  sign_in:
+    type: event
+    description: >
+      A user pressed the sign in button on the sync account page
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_success:
+    type: event
+    description: >
+      A user was successfully signed in to FxA
```

Comment:
Same question as above: what's the purpose of this telemetry?

Reply:
I think the comment above adds the requested information.